sdk/core/core-rest-pipeline/test/tracingPolicy.spec.ts

```diff
@@ -119,9 +124,7 @@ describe("tracingPolicy", function() {
     const request = createPipelineRequest({
       url: "https://bing.com",
       tracingOptions: {
-        spanOptions: {
-          parent: ROOT_SPAN.context()
-        }
+        context: setSpan(context.active(), ROOT_SPAN)
```

Comment:
Can we implement our own `TestContext` that we pass in here instead of using the active context? 

The shape of `Context` looks easy to implement: https://open-telemetry.github.io/opentelemetry-js/interfaces/context.html

Reply:
We could - I didn't think one was too bad and it's mostly a no-op anyways.